build-utils/css/utils/logging.ts

```diff
@@ -0,0 +1,4 @@
+export const logging = (message: string) => {
+  // eslint-disable-next-line no-console
+  console.log(message);
+};
```

Comment:
It might be easier to do something like this: https://stackoverflow.com/a/50101673/15412237

Reply:
I'll have this on my next PR. Thanks